conf/development.ini

```diff
@@ -9,7 +9,6 @@ es.url: http://localhost:9200
 pyramid.debug_all: True
 pyramid.reload_templates: True
 
-h.authority: localhost
```

Comment:
`h.authority` and `sqlalchemy.url` are both needed by the new `init_db.py` script which does not read the Pyramid config file, so I've moved them to envvars in `tox.ini`.